tests/CMakeLists.txt

```diff
@@ -259,9 +263,9 @@ endif()
 #
 message(STATUS "Using C++11")
   if (NOT CMAKE_COMPILER_IS_GNUCXX OR GCC_VERSION VERSION_GREATER 4.6)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11")
```

Comment:
Yes can do. 

The odd thing is that with c++11 you only need D_USE_MATH_DEFINES on Windows. On Linux one of the other switches must turn the constants on. It doesn't make sense to me that math.h behaves differently on different systems when c++11 is selected.